templates/.rubocop.yml

```diff
@@ -30,7 +30,11 @@ Style/FrozenStringLiteralComment:
   
 Style/FormatStringToken:
   Enabled: false  
-
+  
+Style/MixinUsage:
+  Exclude:
+    - 'spec/factories/**'
```

Comment:
Why?

Reply:
if you have something like this

```ruby
include ActionDispatch::TestProcess

FactoryGirl.define do
  factory :company do
    sequence(:name) { |n| "Company #{n} AG" }
    logo { fixture_file_upload(Rails.root.join('spec', 'fixtures', 'test.png')) }
  end
end
```
it will give an error because the `include` is not within a module or a class